xml/System.Data.Common/DbBatch.xml

```diff
@@ -353,10 +406,23 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="cancellationToken">A token to cancel the asynchronous operation.</param>
+        <summary>This is the asynchronous version of <see cref="M:System.Data.Common.DbBatch.ExecuteNonQuery" />. Providers should override with an appropriate implementation. The cancellation token may optionally be ignored.  
```

Comment:
This is also too long for a summary.

Reply:
See https://github.com/dotnet/dotnet-api-docs/pull/7680/files#r836213664